compiler/rustc_mir_build/src/build/coverageinfo.rs

```diff
@@ -106,7 +104,6 @@ fn get_body_span<'tcx>(
 }
 
 fn hash_mir_source<'tcx>(tcx: TyCtxt<'tcx>, hir_body: &'tcx rustc_hir::Body<'tcx>) -> u64 {
-    // FIXME(cjgillot) Stop hashing HIR manually here.
```

Comment:
cc @cjgillot on removing this FIXME.

I believe it was mostly obsoleted by #95573, since this code now just fetches an already-computed hash, instead of computing a hash manually.

Now that this code runs during MIR building (instead of MIR transform), it looks like there's nothing left to fix here.